system.md

```diff
@@ -0,0 +1,379 @@
+# Terra System's Rhûn {#system}
+
+This section details how to play **Shadows of Rhûn**, a [The One Ring](https://freeleaguepublishing.com/games/the-one-ring/) inspired *Play by Post* RPG using a forked version of the **Terran Engine v0.6.2**.
+The **Terran Engine** is self contained in this section, and will not require reading the [SRD](https://efsalvarenga.github.io/terraSystem/) separately.
+
+#### Taxonomy {-}
+
+- "You"/"Your" are used to refer to players and their characters, interchangeably.
+- "Challenge" is used to represent anything that is antagonistic to you, be that an NPC, the environment, etc.
+- Meta game concepts, stats and currencies are marked in *italic*.
+- Book references are marked in `code`, they might be accompanied by page ranges.
+
+## Characters
+
+1. With the other players, define a cohesive group concept.
+2. Always start with a Name and *Archetype*.
+3. Roll for *Attributes* and assign points.
+4. Roll for *Grit* and *Ego.*
+5. Choose one *Advancement.*
+6. Make an image.
+
+A text version of the character sheet is available here.
+
+### Archetypes
+
+You are defined by an *Archetype*, which encapsulates your background, concept and role within the narrative.
+Each *Archetype* should offer an immediate grasp of your identity and behaviours.
+It is expected you will evolve beyond these initial *Archetypes* towards unique narratives.
+
+< ADAPT TABLE BELOW FOR RHUN>
```

Comment:
Do you plan to push the adaptation as part of this PR or a new one?

Reply:
As a later one. I prefer to do small deltas. And you can have a look as well. I plan on adapting from stuff on The One Ring. So we wait for your books to arrive.